src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,88 @@
+<br/><br/>
+<img src="./send.png" width="400" alt="Send field for crypto address"/>
+<br/><br/>
+
+4. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are. 
+5. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 
+6. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.
+
+## Connecting to projects
+
+Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.
+
+1. Visit any project’s website.
+2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 
+3. Once you are in the app click on “Connect”
+
+<br/><br/>
+<img src="./connect1.png" width="400" alt="Button allowing user to connect to the website with a wallet"/>
+<br/><br/>
+
+4. Select your wallet from the offered list. If you can't see your wallet, maybe it will be hidden under “WalletConnect” option.
+
+&nbsp;
+<img src="./connect2.png" width="400" alt="Selecting from a list of wallets to connect with"/>
+&nbsp;
+
+5. Confirm a signature request in your wallet to start the connection. It should not cost anything.
+6. Thats it! Start using the app. You can find some interesting projects on our [dApps page](https://ethereum.org/en/dapps/#explore).
+
+<br/><br/>
+
+### FAQ
+
+**If I own an ETH address, do I own the same address on other blockchains?**
+
+- You can use the same address on many popular blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.
+
+**Can I use the same address on multiple devices?**
+
+- Yes you can use the same address on multiple devices. Wallets are technically only an interface to show you your balance and to make transactions, your wallets are not store inside the wallet, but on the blockchain.
+
+**I have not received the crypto, where can I check the status of a transaction?**
+
+- You can use [block explorers](https://ethereum.org/en/developers/docs/data-and-analytics/block-explorers/) to see the status of any transaction in real time. All you need to do is to search your wallet address or the ID of the transaction.
+
+**Can I cancel or return transactions?**
+
+- No, once transaction is initiated it cannot be canceled.
```

Comment:
```suggestion
- No, once a transaction is initiated, you can not cancel the transaction.
```

Reply:
more caveats! technically you can cancel a transaction that is pending (e.g. if you sent too low a gas fee). however, if it's a successful transaction, then saying this makes more sense (let's ignore the complexities of that specific case here 😅)